lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -178,23 +178,35 @@ def get_rotate_label(self, text):
             return len(text) > 4
 
     def _get_coord_info(self, renderer):
-        mins, maxs = np.array([
-            self.axes.get_xbound(),
-            self.axes.get_ybound(),
-            self.axes.get_zbound(),
-        ]).T
-        centers = (maxs + mins) / 2.
-        deltas = (maxs - mins) / 12.
-        mins = mins - deltas / 4.
-        maxs = maxs + deltas / 4.
-
-        vals = mins[0], maxs[0], mins[1], maxs[1], mins[2], maxs[2]
-        tc = self.axes.tunit_cube(vals, self.axes.M)
-        avgz = [tc[p1][2] + tc[p2][2] + tc[p3][2] + tc[p4][2]
-                for p1, p2, p3, p4 in self._PLANES]
-        highs = np.array([avgz[2*i] < avgz[2*i+1] for i in range(3)])
-
-        return mins, maxs, centers, deltas, tc, highs
+        mins, maxs = np.array(
```

Comment:
We don't usually add the extra lines per-bracket like this.

Reply:
This is just what `black` suggests which is triggered every time I save the file in my IDE. I can switch it back, but I feel like matplotlib is swimming against the current here. 

There are so many larger python projects (pandas, dask, xarray etc) that uses it now that you likely will waste more and more time with these kinds of code style discussions.

But matplotlib uses a lot of linear algebra and matrix calculations which I think `black` is frankly terrible at, so I can understand the reluctance of relying on it completely.

Reply:
We don't use black, see https://github.com/matplotlib/matplotlib/issues/18796 (your point that black is particularly bad at math expressions actually summarizes our problem quite well :-)).